bound/kt/src/test/kotlin/web5/sdk/dids/BearerDidTest.kt

```diff
@@ -1,41 +1,41 @@
 package web5.sdk.dids
 
-import org.junit.jupiter.api.Assertions.assertEquals
-import org.junit.jupiter.api.Assertions.assertNotNull
-import org.junit.jupiter.api.Test
-import web5.sdk.crypto.keys.InMemoryKeyManager
-import web5.sdk.crypto.keys.Jwk
-import web5.sdk.dids.methods.jwk.DidJwk
-import web5.sdk.rust.ed25519GeneratorGenerate
+//import org.junit.jupiter.api.Assertions.assertEquals
+//import org.junit.jupiter.api.Assertions.assertNotNull
+//import org.junit.jupiter.api.Test
+//import web5.sdk.crypto.keys.InMemoryKeyManager
+//import web5.sdk.crypto.keys.Jwk
+//import web5.sdk.dids.methods.jwk.DidJwk
+//import web5.sdk.rust.ed25519GeneratorGenerate
 
 class BearerDidTest {
 
-    @Test
-    fun `test basic bearer did creation`() {
-        val privateJwk = ed25519GeneratorGenerate()
-
-        val keyManager = InMemoryKeyManager(listOf())
-        val publicJwk = keyManager.importPrivateJwk(Jwk.fromRustCoreJwkData(privateJwk))
-
-        val didJwk = DidJwk(publicJwk)
-
-        val bearerDid = BearerDid(didJwk.did.uri, keyManager)
-
-        assertEquals(bearerDid.document.id, didJwk.document.id)
-    }
-
-    @Test
-    fun `test bearer did sign`() {
-        val privateJwk = ed25519GeneratorGenerate()
-
-        val keyManager = InMemoryKeyManager(listOf())
-        val publicJwk = keyManager.importPrivateJwk(Jwk.fromRustCoreJwkData(privateJwk))
-
-        val didJwk = DidJwk(publicJwk)
-
-        val bearerDid = BearerDid(didJwk.did.uri, keyManager)
-
-        val signedPayload = bearerDid.getSigner().sign("hi".toByteArray())
-        assertNotNull(signedPayload)
-    }
+//    @Test
```

Comment:
oh wait maybe you just forgot to uncomment these tests

Reply:
nah it's because I'm trying to do the least amount of temporary things, and the bearer did api surface will change in the next few days. I'll just remove this code altogether